doc/release-notes/release-7.sgml.in

```diff
@@ -221,6 +221,14 @@ This section gives an account of those changes in three categories:
 	<tag>--enable-cachemgr-hostname=</tag>
 	<p>The <em>cachemgr.cgi</em> tool this option relates to has been removed.
 
+	<tag>--disable-ident-lookups</tag>
+	<p>The primary purpose of keeping Ident lookup feature behind a ./configure
+	option was avoiding performance impact of the corresponding code when the
+	feature was unused. That goal is now achieved without that complication.
```

Comment:
False. "Primary" purpose of all `--disable-foo` options is to allow distributors or admin to prohibit use of components for *any reason they desire*. Nothing more, nothing less.

For this specific `--disable-ident-option`, security is more relevant. Being able to globally lock down Ident Protocol use by end-user configured proxies is a major need for certain types of CDN installation. Firewall rules cannot go far enough to block the security hole they face.

Please restore this build option. It is needed so long as Squid contains code supporting Ident natively.